documentation/book/proc-accessing-kafka-using-routes.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='proc-accessing-kafka-using-routes-{context}']
+= Accessing Kafka using {OpenShiftName} routes
+
+.Prerequisites
+
+* An {OpenShiftName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Deploy Kafka cluster with external listener enabled and configured to the type `route`.
```

Comment:
Deploy a Kafka cluster with an external listener enabled and configured with the type `route`.